src/test/java/org/jabref/logic/pdf/PdfAnnotationImporterTest.java

```diff
@@ -45,14 +47,21 @@ public void noAnnotationsMinimal() {
     @Test
     public void inlineNoteMinimal() {
         List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/minimal-inlinenote.pdf"));
-        assertEquals(1, annotations.size());
 
+        // Begin old test
+        assertEquals(1, annotations.size());
         FileAnnotation note = annotations.get(0);
         assertEquals("Linus Dietz", note.getAuthor());
+        assertEquals(LocalDateTime.of(2017, 3, 12, 20, 25), note.getTimeModified());
+        assertEquals(1, note.getPage());
         assertEquals("inline note annotation", note.getContent());
-        assertFalse(note.hasLinkedAnnotation());
         assertEquals("FreeText", note.getAnnotationType());
-        assertEquals(1, note.getPage());
+        assertFalse(note.hasLinkedAnnotation());
+        // End old test
+
+        FileAnnotation expected = new FileAnnotation("Linus Dietz", LocalDateTime.of(2017, 3, 12, 20, 25), 1, "inline note annotation", "FreeText", Optional.empty());
+        assertEquals("Comparison fails. WHY?", Collections.singletonList(expected), annotations);
```

Comment:
Not directly to see, but I did some debuggin and it  fails because the FileAnnotation does not provide an equals method